sympy/integrals/transforms.py

```diff
@@ -1284,6 +1285,10 @@ def _fourier_transform(f, x, k, a, b, name, simplify=True):
     For suitable choice of a and b, this reduces to the standard Fourier
     and inverse Fourier transforms.
     """
+
+    if limit(f, x, -oo) != 0 and (limit(f, x, -oo) is not S.NaN or limit(f, x, -oo).is_number or limit(f, x, -oo) == S.Infinity or limit(f, x, -oo) == S.NegativeInfinity):
```

Comment:
Why don't you store the return value of the limit function in a variable, which you can use in the condition for the if statement? Please try to make the condition more readable by using multiple lines for the if statement.